.github/workflows/tests.yml

```diff
@@ -100,10 +111,18 @@ jobs:
                 pip install . -v
             fi
 
+          if [ "${OPTIONS_NAME}" == "editable-install" ]; then
+              echo "Testing an editable installation of pywt"
+              sudo apt-get install ninja-build
+              pip install numpy>=2.0.0b1 cython>=3.0.4 meson-python>=0.16.0 matplotlib pytest --force-reinstall
+              pip install -e . --no-build-isolation
```

Comment:
I had to keep `--force-reinstall` here because it was otherwise using `numpy<2` to compile (which shouldn't be a problem, but good to stay updated since 1.6.0 uses the RC). The workflow matrix is a bit complex so this was the best way that came to mind, I would be happy to explore more ways (probably best left for another PR later on?).

Reply:
`numpy` and `cython` are already installed, so I'd prefer just dropping them here. and then you can also drop `--force-reinstall`. There is no need to reinstall, and I'm fine with just testing with the latest stable release.